### DISCUSSION TITLE
PrimeFaces JSF Designer Not Working

### DISCUSSION THREAD

**Discussion by simonjcarr** [primefaces/community#2315] [PrimeFaces] [answered]
### Describe the bug

https://www.primefaces.org/designer-jsf/ is not working. I just get the following

![image](https://github.com/user-attachments/assets/a06c58b1-4728-4a39-8dcf-a4af3ff1cb90)

### Reproducer

Visit https://www.primefaces.org/designer-jsf/

### Expected behavior

After clicking a base theme I would be taken to a designer screen.

### PrimeFaces edition

None

### PrimeFaces version

xxx

### Theme

all

### JSF implementation

All

### JSF version

xx

### Java version

xx

### Browser(s)

Chrome and Edge

**Comment by melloware** [comment]
i have moved your issue to to the PrimeFaces Community page.  PF gitHub is only for defects related to PF.

**Comment by thomsa** [comment]
This is still a problem. Would be nice to be able to use the theme builder.

**Reply by melloware** [reply]
PrimeTek is retiring this designer

**Reply by contis2908** [reply]
This seems like a crucial information. All the links are broken and nowhere is stated that they discontinued it.
I find information about them making it open-source but not retiring it.

**Reply by melloware** [reply]
@cagataycivici can you add some clarity here?

**Comment by zahraa-hareb** [comment]
Facing the same problem..

**Reply by melloware** [reply]
PrimeTek is removing the designer.

**Comment by cagataycivici** [comment, marked as answer]
The theme designer has been shut down. Please use the sass files for theming.

https://github.com/primefaces/primefaces-sass-theme

**Comment by matsandreassen-triona** [comment]
Searches still result in finding the theme designer pages with no notice about discontinuation. Also, licenses can be purchased in your store. 

Since we used to have a license to this tool, I got so far to have purchased a new license (and sent you a refund request). Maybe you could take another swipe at removing the tool's online presence?

Also, starting from the "Theming" page in the showcase, it's a bit vague which SASS files I should base my theme on. The primefaces-sass-theme repo seems to have been deprecated with a link to the main repository, but since we're still on the PF 14 branch, should I use the theme repo or not?

I might have misunderstood something? Even more than one thing?
